CODE_OF_CONDUCT.md

```diff
@@ -0,0 +1,22 @@
+# The Solidus Code of Conduct
+
+Solidus strives to be inclusive of as many contributors as possible.
+
+All contributors and participants are welcome regardless of gender, sexual orientation, ability, race, ethnicity, socioeconomic status, and religion (or lack thereof).
+
+This document highlights the values of the Solidus community. It includes our expectations for community members. It is inspired by elements of the [Ruby Community Conduct Guideline](https://www.ruby-lang.org/en/conduct/).
+
+It applies to all "collaborative space", which is defined as community communications channels (such as mailing lists, messaging platforms, conferences, meetups, submitted patches, commit comments, etc.).
+
+## Community Expectations
+
+- Participants will be tolerant of opposing views and new ideas.
+- Participants will not engage in personal attacks and disparaging personal remarks.
+- When interpreting the words and actions of others, participants will always assume good intentions.
+- Behavior which can be considered harassment will not be tolerated.
```

Comment:
I'd also add:

> - Partecipants should criticize, judge or blame ideas rather than people behind them

which in my opinion is very appropriate for the kind of interactions that currently happens in our project.